packages/ember-routing-htmlbars/lib/keywords/action.js

```diff
@@ -7,7 +7,7 @@ import { keyword } from 'htmlbars-runtime/hooks';
 import closureAction from 'ember-routing-htmlbars/keywords/closure-action';
 
 /**
-  The `{{action}}` helper provides a way to pass triggers for behavior (usually
+  The `{{action` helper provides a way to pass triggers for behavior (usually
```

Comment:
Agreed on both points.

@locks @tomdale Agreed that I should switch `{{action` to `{{action}}` for this PR? Any other changes for this one?